app/src/main/java/com/github/amusinamaria/MoviesApp.kt

```diff
@@ -0,0 +1,7 @@
+package com.github.amusinamaria
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
```

Comment:
For what?

Reply:
I'll use it after merge (in a branch with viewmodel)